src/app/components/client/DataBrokerProfiles.tsx

```diff
@@ -10,12 +10,20 @@ import { useL10n } from "../../hooks/l10n";
 import IconChevronDown from "./assets/icon-chevron-down.svg";
 import { useState } from "react";
 import { OnerepScanResultRow } from "knex/types/tables";
-import { OpenInNew } from "../server/Icons";
+import { DataBrokerNames } from "../../functions/universal/dataBrokerNames";
 
 export type Props = {
   data: OnerepScanResultRow[];
 };
 
+function getDataBrokerName(dataBrokerName: string) {
```

Comment:
Since `DataBrokerNames` is located in `/functions/universal/`, might it make sense to move this function in there?

(Also, it's simple enough to not need it, but if you'd like practice writing unit test... This is basically the perfect scope to give it a shot :smile: Could be one test to verify that it comes up with the pretty name if available, and one that it fallbacks to the URL if not.)

Reply:
(And in fact, if you do change it into a record as per my other comment, you could even write the test beforehand to verify that the refactor doesn't break anything :grin:)

Reply:
Ha. I did exactly this. Landed the test (da34a229cce53670f84372b56208e99669c4f22c) and then updated to the record lookup (dec693c9bf6f565a1418d112222304840446b72e)